python/utils.py

```diff
@@ -545,9 +545,12 @@ def reloadProjectMacros():
         return
 
     # create a new empty python module
-    import imp
-
-    mod = imp.new_module("proj_macros_mod")
+    if sys.version_info >= (3, 4):
+        import importlib
```

Comment:
Could we use that unconditionally? It should be around since 3.1

Reply:
Sure, I just wanted to be safe in case someone still using old Python.